nanome/_internal/_structure/_helpers/_copy.py

```diff
@@ -1,49 +1,87 @@
 def _deep_copy_complex(complex):
```

Comment:
So this (with the shallow copy changes) is the real heart of this fix.
I was rewriting to_frames and I realized that the new algorithm looked a LOT like deep_copy. So instead, I added a "conformer number" to the deep_copy. 
This is not a feature that API users can access (yet).

Here's how this works.
If you don't pass a conformer_number, you get an exact replica of your original structure.
If you pass in a conformer number, then the algorithm will return that structure but only the subtree for that conformer.
If you pass a conformer number, and the structure does not exist on that frame (residue with no atoms, atom that's disabled etc...), it returns None.
Molecules exist on all frames even if there are no atoms because they still have associateds.